.github/workflows/pages.yml

```
@@ -26,10 +26,31 @@ jobs:
         with:
           version: 8
           
+      - name: Cache pnpm store
+        uses: actions/cache@v4
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('website/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)